hieradata_aws/class/production/backend.yaml

```diff
@@ -3,7 +3,5 @@
 icinga::client::check_pings::endpoints:
    whitehall-admin:
      ip: 10.3.3.254
-   content-performance-manager:
-     ip: 10.3.3.254
```

Comment:
I've looked in to this now, and this relates to checking the connectivity between AWS and Carrenza. As we're not running the Content Performance Manager in Carrenza any longer, I guess the check may start to fail due to the other changes on this branch.

As we're trying to also connect to Whitehall Admin, on the same IP address, I don't think there's any need to replace this, it can just be removed.